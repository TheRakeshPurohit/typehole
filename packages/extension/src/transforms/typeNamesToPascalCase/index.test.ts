import { typeNamesToPascalCase } from ".";

test("finds all dependency type declarations from an ast when given one interface", () => {
  const typeString = `
  interface AutoDiscovered {
    available_tenants: IAvailable_tenants;
  }

  interface IAvailable_tenants {
    primary: IPrimary;
  }`;
  const result = typeNamesToPascalCase(typeString);

  expect(result).toMatchSnapshot();
});
