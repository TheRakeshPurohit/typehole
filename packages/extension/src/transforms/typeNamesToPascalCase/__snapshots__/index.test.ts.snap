// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`finds all dependency type declarations from an ast when given one interface 1`] = `
"interface AutoDiscovered {
    available_tenants: IAvailableTenants;
}
interface IAvailableTenants {
    primary: IPrimary;
}
"
`;
